api/api.go

```diff
@@ -93,31 +92,32 @@ func (srv *Server) initAPI(addr string) {
 		handleHTTPRequest(mux, "/renter/status", srv.renterStatusHandler)
 	}
 
-	// TransactionPool API Calls
+	// TransactionPool API Calls - Unfinished
 	if srv.tpool != nil {
 		handleHTTPRequest(mux, "/transactionpool/transactions", srv.transactionpoolTransactionsHandler)
 	}
 
 	// Wallet API Calls
 	if srv.wallet != nil {
-		handleHTTPRequest(mux, "/wallet/address", srv.walletAddressHandler)
-		// handleHTTPRequest(mux, "/wallet/merge", srv.walletMergeHandler)
-		handleHTTPRequest(mux, "/wallet/send", srv.walletSendHandler)
-		// handleHTTPRequest(mux, "/wallet/status", srv.walletStatusHandler)
-		handleHTTPRequest(mux, "/wallet/siafunds/balance", srv.walletSiafundsBalanceHandler)
-		handleHTTPRequest(mux, "/wallet/siafunds/send", srv.walletSiafundsSendHandler)
-		handleHTTPRequest(mux, "/wallet/siafunds/watchsiagaddress", srv.walletSiafundsWatchsiagaddressHandler)
+		handleHTTPRequest(mux, "/wallet", srv.walletHandler)                          // GET
+		handleHTTPRequest(mux, "/wallet/close", srv.walletCloseHandler)               // PUT
+		handleHTTPRequest(mux, "/wallet/history", srv.walletHistoryHandler)           // GET, $(addr) GET
+		handleHTTPRequest(mux, "/wallet/seed", srv.walletSeedHandler)                 // GET, PUT, POST
+		handleHTTPRequest(mux, "/wallet/siacoins", srv.walletSiacoinsHandler)         // PUT
+		handleHTTPRequest(mux, "/wallet/siafunds", srv.walletSiafundsHandler)         // PUT
+		handleHTTPRequest(mux, "/wallet/transaction", srv.walletTransactionHandler)   // $(id) GET
+		handleHTTPRequest(mux, "/wallet/transactions", srv.walletTransactionsHandler) // GET
+		handleHTTPRequest(mux, "/wallet/unlock", srv.walletUnlockHandler)             // PUT
 	}
 
-	// BlockExplorer API Calls
+	// BlockExplorer API Calls - Unfinished
 	if srv.exp != nil {
 		handleHTTPRequest(mux, "/explorer/status", srv.explorerStatusHandler)
 		handleHTTPRequest(mux, "/explorer/blockdata", srv.explorerBlockDataHandler)
 		handleHTTPRequest(mux, "/explorer/gethash", srv.explorerGetHashHandler)
-
 	}
 
-	// create graceful HTTP server
+	// create graceful HTTP server - Unfinished
```

Comment:
unfinished, eh?

I don't see why the wallet is supposedly "finished" but no other modules are.